boards/sim/sim/sim/scripts/Make.defs

```diff
@@ -267,6 +267,10 @@ else
   LDFLAGS += -Wl,-Ttext-segment=0x400000
 endif
 
+ifeq ($(CONFIG_DEBUG_LINK_MAP),y)
+  LDFLAGS += -Wl,-Map=$(TOPDIR)/nuttx.map
```

Comment:
Will that work in case of CONFIG_HOST_MACOS? I just never tried this, so asking :)

Reply:
done